spec/07-variables.md

```diff
@@ -259,6 +281,8 @@ function f()
 }
 ```
 
+Be also aware that declaring a variable global can hide a local variable and/or a function static with the same name. See ([§§](#hidingNotice)) for an example.
```

Comment:
Hi Stas, thanks for the feedback, the naming `function static` is used in the file and thus it does not make sense to introduce a new naming (`static variable`) IMO. The pull request is more than one year old, these days it was common to use [§§]. I am not going to rebase my branch and do the change, it is just too much hassle to bring a small pull request into the spec without commit rights. I leave it to someone with commit rights to do the change, because guess what... if I do the change it might well be that I need to wait another month until someone decides to merge it or someone has some additional feedback then I would need to wait even longer. 

My advice to people with commit rights in general: merge a pull request which adds missing information to the spec and improve it with additional pull requests